api_client/project_service_client.py

```diff
@@ -17,10 +17,13 @@ def __init__(self, session: Session, service_base_url):
         self.session = session
         self.service_base_url = service_base_url
 
-    def run_analysis(self, analysis_id):
-        url_path = f'/project/v1/analyses/{analysis_id}/jobs'
+    def run_analysis(self, analysis_id, with_attr = None):
+        if with_attr:
+            url_path = f'/project/v1/analyses/{analysis_id}/jobs?withAttribution=true'
+        else:
+            url_path = f'/project/v1/analyses/{analysis_id}/jobs'
         url = urllib.parse.urljoin(self.service_base_url, url_path)
-
+        logging.info(f'now making run analysis call with url: {url}')
```

Comment:
Please make sure both the cases and apic command line are tested to make sure backward compatiblity